src/pages/docs/chat/api/javascript/occupancy.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: Occupancy
+meta_description: ""
+---
+
+The `Occupancy` interface is used to subscribe to occupancy updates and fetch the occupancy metrics of a room.
+
```

Comment:
For this, and for other features - we should definitely discuss what room options you need to set to make them work, on this page